portal-api/src/scheduler/jobs/send-reporting-period-start-emails-job/index.js

```diff
@@ -0,0 +1,69 @@
+const api = require('../../../v1/api');
+const sendEmail = require('../../../external-api/send-email');
+const EMAIL_TEMPLATE_IDS = require('../../../constants/email-template-ids');
+const { hasValue, isValidEmail } = require('../../../utils/string');
+const { getReportPeriod, getReportDueDate } = require('./helpers');
+
+const FIRST_OF_MONTH_AT_7_AM_CRON = '0 7 1 * *';
```

Comment:
When you implement the next ticket of checking every day are you going to replace this with just a daily cron job? Or are you expecting multiple cron jobs for each type of email and they all run separately/back to back? FWIW I think I'd lean towards the former but happy to discuss!

Reply:
I was planning to have them as separate jobs. We will only need to have a cron expression for every day for the other two reports because you can't configure cron to run on a certain business day. If all the jobs could be run on specific dates we would have had three separate jobs just running on the specific date. It didn't feel right to have the other two reports run on the same job just because they happen to have the same schedule. It didn't feel like there was much benefit from doing that, and it may set a precedent where more unrelated code is added to the same job just because the check needed to be done every day. What do you think?

Reply:
After reading this I've been convinced, even after we've discussed how it doesn't matter it's all on the same server I was still trying to save compute time in my head aha. Happy to go with what you've said here